maps/simid_protocol.js

```diff
@@ -0,0 +1,388 @@
+/**
+ * Contains logic for sending mesages between the SIMID creative and the player.
+ * Note: Some browsers do not support promises and a more complete implementation
+ *       should consider using a polyfill.
```

Comment:
Add a note giving the url where this file originally came from.

Reply:
Added